docs/source/analyst/worked_examples/index.md

```diff
@@ -8,8 +8,7 @@ We have five worked examples of using FlowKit for CDR analysis:
 - [Commuting Patterns](commuting-patterns.ipynb),
```

Comment:
(I can't add a comment on line 5, so I'll add it here)

"We have five worked examples of using FlowKit for CDR analysis:" -> "We have four worked examples of using FlowClient for CDR analysis:"